near-sdk/src/types/event.rs

```diff
@@ -0,0 +1,84 @@
+pub trait StandardEvent {
+    type EventString: std::fmt::Display;
```

Comment:
Wondering if anyone else has opinions, but having this named as `EventJson` might be more clear?

Honestly not thrilled that our events and API is being forced as JSON, and we don't have much flexibility to use a more efficient format in the future, but this would make it more clear what this format is?

wdyt @frol ?

Reply:
Yeah, naming is hard. The standard uses `EVENT_JSON:` prefix specifically to keep the opportunity open for future extension. Thus, `EventJson` makes total sense to me as it matches the prefix. Maybe even consider renaming `#[near_bindgen(event)]` to `#[near_bindgen(event_json)]`?

Reply:
I don't want to drag ourselves into the conversation of how other formats could look like - borsh does not have a schema, and binary data cannot be dumped into UTF-8 string, so before getting dumped into logs, we either would need to make a protocol change in NEAR Protocol that would turn the logs to be an array of bytes instead of a valid UTF-8, or encode borsh-serialized event into, just for the sake of example, base64.

Thus, I gravitate towards actually renaming all the mentions of just `event` into `event_json` and get back to this discussion later, when we will have several events standards.

Reply:
should I move forward with @frol recommendation?

Reply:
I don't have an opinion if you're asking me. What frol suggested SGTM :)